server/cluster-whitelist.js

```diff
@@ -8,18 +8,25 @@ module.exports = {
       roxie_port: "18002",
     },
     {
-      name: "4-Way",
+      name: "4-Way-2",
       thor: "http://10.173.147.1",
       thor_port: "8010",
       roxie: "http://10.173.147.1",
       roxie_port: "8002",
     },
     {
-      name: "4-Way-2",
-      thor: "http://10.173.147.1",
-      thor_port: "8010",
-      roxie: "http://10.173.147.1",
-      roxie_port: "8002",
+      name : "new",
+      thor: "http://52.255.183.194",
```

Comment:
We should not be checking in this file with internal IP's. This should just be a place holder and during the application setup, it will changed to include the respective clusters

Reply:
added file to git ignore